packages/foreman/rubygem-puma/rubygem-puma.spec

```diff
@@ -0,0 +1,101 @@
+# FIXME:
+#   1. Change scl_prefix to _ruby or _ror for Requires/BuildRequires that are
+#      in Ruby or Rails software collections (not Foreman's)
+#   2. Delete these lines
+#
+# Generated from puma-3.11.4.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name puma
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 3.11.4
+Release: 1%{?dist}
+Summary: Puma is a simple, fast, threaded, and highly concurrent HTTP 1.1 server for Ruby/Rack applications
+Group: Development/Languages
+License: BSD-3-Clause
```

Comment:
The Fedora guidelines prefer `BSD` according to https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses (and maybe we'll need a SPDX-to-Fedora mapping in gem2rpm).